workspace/__lib__/xod/i2c/request/patch.xodp

```diff
@@ -0,0 +1,76 @@
+{
+  "description": "Requests N bytes from a given slave I²C device",
+  "nodes": [
+    {
+      "description": "Slave device I²C address to request from",
+      "id": "BkRw4E3kX",
+      "label": "ADDR",
+      "position": {
+        "x": 68,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-byte"
+    },
+    {
+      "description": "I2C interface object",
+      "id": "Hk1dRQh17",
+      "label": "I2C",
+      "position": {
+        "x": 0,
+        "y": 0
+      },
+      "type": "@/input-i2c"
+    },
+    {
+      "description": "Pulses if request failed. E.g., due to an invalid `ADDR`.",
+      "id": "S1eJu0Q2JQ",
+      "label": "ERR",
+      "position": {
+        "x": 68,
+        "y": 204
+      },
+      "type": "xod/patch-nodes/output-pulse"
+    },
+    {
+      "description": "Sends the request",
+      "id": "SJjmdOVOW",
+      "label": "SEND",
+      "position": {
+        "x": 204,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-pulse"
+    },
+    {
+      "description": "Pulses when the request is successfully send. Send it to a `read-byte`.",
+      "id": "Sk7BudNdZ",
+      "label": "DONE",
+      "position": {
+        "x": 0,
+        "y": 204
+      },
+      "type": "xod/patch-nodes/output-pulse"
+    },
+    {
+      "boundLiterals": {
+        "__out__": "01h"
+      },
+      "description": "Number of bytes to request",
+      "id": "r1NONEn1m",
+      "label": "N",
+      "position": {
+        "x": 136,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-byte"
```

Comment:
Is there a reason this is an `input-byte` besides https://github.com/arduino/ArduinoCore-avr/blob/b7c607663fecc232e598f2c0acf419ceb0b7078c/libraries/Wire/src/Wire.h#L61-L63 ?
IMHO, "**Number** of bytes to request" should be a number.

Reply:
Oops, no. Good catch. Should be a number, I will fix it